src/pages/DetailsPage.js

```diff
@@ -148,7 +149,9 @@ class DetailsPage extends React.PureComponent {
                                 </AttachmentModal>
                                 {details.displayName && (
                                     <Text style={[styles.textHeadline, styles.mb6]} numberOfLines={1}>
-                                        {isSMSLogin ? this.props.toLocalPhone(details.displayName) : details.displayName}
+                                        {isSMSLogin
+                                            ? this.props.toLocalPhone(details.displayName)
+                                            : renderTextWithEmoji(details.displayName, {...styles.emojiMessageText, ...styles.profileEmojiText})}
```

Comment:
Passing props to the render function looks unusual if the render function is imported outside. Maybe it's better to make a function component something like <TextEmoji>? wdyt @josemak25 ?

Reply:
I think this a TextEmoji works better, I would make these change right away